api/python/provisioner/salt_api/base.py

```diff
@@ -259,8 +260,8 @@ def run(
 
         try:
             logger.debug(
-                f"'{type(self)}' client: Function '{fun}' resulted in "
-                f"{res.results}"
+                "'{}' client: Function '{}' resulted in: {}"
```

Comment:
Why have we moved away from f-string?

Reply:
@ypise only in places where I've used pretty format `pformat`, I've used `{}.format()'. Changed to f-string in other places.